Libraries/Hotcakes.Commerce.NoCms/Hotcakes.Commerce.NoCms.csproj

```diff
@@ -58,9 +58,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\References\Dnn\System.Web.Helpers.dll</HintPath>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=4.0.0.1, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Microsoft.AspNet.Mvc.4.0.40804.0\lib\net40\System.Web.Mvc.dll</HintPath>
+    <Reference Include="System.Web.Mvc">
```

Comment:
Why was the MVC upgrade necessary?

Reply:
i have reverted above change, when i have checkout project and try to build but it failed due to System.Web.Mvc.dll was missing so,  i have used  reference from nuget packages folder has latest version of MVC DLL(Microsoft.AspNet.Mvc.5.1.1) so, i have added reference of same.